subcommands/devices/config_updates.go

```diff
@@ -21,12 +21,31 @@ currently configured and reporting.`,
   # Make a device start taking updates from Targets tagged with "devel"
   fioctl devices config updates <device> --tag devel
 
-  # Set the docker apps a device will run:
+  # Set the Compose apps a device will run:
   fioctl devices config updates <device> --apps shellhttpd
 
-  # Set the docker apps and the tag:
+  # Set the Compose apps and the tag:
   fioctl devices config updates <device> --apps shellhttpd --tag master
-`,
+
+	# There are two special characters: "," and "-"
+	# The following explains what happens if you provide a ","
+  # Set the Compose apps to none, meaning it will run no apps:
```

Comment:
```suggestion
  # There are two special characters: "," and "-".
  # Providing a "," sets the Compose apps to "none", meaning it will run no apps:
```